odbcconfig/odbcinst.ini.template

```diff
@@ -0,0 +1,7 @@
+[ODBC]
+#Trace=yes
+#TraceFile=~/odbc.trace
+
+[EXAODBC]
+#Driver location needs to be added in build environment:
+DRIVER=<PATH_TO_DRIVER>
```

Comment:
This template is used by a user and not a program or? If it would be used by a program, we should use Jinja or so

Reply:
User